deploy-as-code/helm/environments/unified-uat.yaml

```diff
@@ -3,158 +3,176 @@ global:
    setup: fullsetup
 
 cluster-configs:
-    namespaces:  
-      create: true
-      values: [ backbone, cert-manager, egov, es-cluster, es-upgrade, kafka-cluster, logging, monitoring, playground, zookeeper-cluster ]
+  namespaces:  
+    create: true
+    values: [ backbone, cert-manager, egov, es-cluster, kafka-cluster, logging, monitoring, playground, pms ]
```

Comment:
can u check this pr, connect with debashish, apply change only for public service related, this will cause environment collapse issue